code/__HELPERS/roundend.dm

```diff
@@ -520,60 +538,79 @@
 	log_econ("Roundend service income: [tourist_income] credits.")
 	switch(tourist_income)
 		if(0)
-			parts += "<span class='redtext'>Service did not earn any credits...</span><br>"
+			parts += "<span class='neutraltext'>service did not interact with the tourist program enough for a grade.</span><br>"
```

Comment:
lowercase intentional?

Reply:
Yes, before these announcements had switching caps and that's bad